ibis/backends/dask/tests/conftest.py

```diff
@@ -14,7 +14,10 @@
 
 dd = pytest.importorskip("dask.dataframe")
 
-NPARTITIONS = 2
+# FIXME Force npartitions=1 because of a dask issue with non deterministic groupby results.
+# The issue relates to the shuffle method on a local cluster, using npartitions=1 in tests avoids it.
+# https://github.com/dask/dask/issues/10034
+NPARTITIONS = 1
```

Comment:
We need `NPARTITIONS` to be greater than one so as to not mask cross partitioning bugs. 

Can you use the alternative fix suggested for the jira for any tests where we are getting bugs from non determinism before?